include/glow/Graph/Graph.h

```diff
@@ -46,6 +46,14 @@ using PlaceholderList = std::list<Placeholder *>;
 using UnsignedArrayRef = llvm::ArrayRef<size_t>;
 /// Map from original Nodes to cloned Nodes.
 using NodeMap = llvm::DenseMap<Node *, Node *>;
+/// State of a function.
```

Comment:
Please provide a bit more elaborative description for `FunctionState` and how/where it is supposed to be used.